wger/exercises/views/exercises.py

```diff
@@ -57,13 +59,21 @@
     TranslatedSelectMultiple,
     TranslatedOriginalSelectMultiple
 )
+from wger.exercises.api.serializers import ExerciseSerializer
 from wger.config.models import LanguageConfig
 from wger.weight.helpers import process_log_entries
 
 
 logger = logging.getLogger(__name__)
 
 
+class ExercisesListView(APIView):
```

Comment:
@Manorlds-Eaglespark this class seems to be a duplicate to the already existent  `class ExerciseListView(APIView).

Reply:
Thanks for your comment, kindly read the pivotal tracker story carefully, the aim of the task is to list all exercises with details like muscles, category, images so a client can see all the available exercises in their detail info, in one place.      I made a new class for this endpoint because this feature is required to have a unique endpoint. Kindly confirm this. thanks.